public/index.html

```diff
@@ -29,32 +29,49 @@
       </div>
 
       <div class="content">
-        <h2>Hey there!</h2>
-
-        <p>Let's jump right in. What's your favourite color?</p>
 
-        <form id="formColor">
-          <p>
-            <input class="form-control" placeholder="e.g. green">
-          </p>
-          <p>
-            <button type="submit" class="btn btn-primary form-control">submit</button>
-          </p>
-        </form>
+        <h2>Hey there!</h2>
 
-        When you click submit, this is what happens:
+        <p>
+          Let’s jump right in: this entire tutorial is being downloaded with
+          the code below. Your ServiceWorker status is:
+          <strong class="service-worker-status"></strong>
+        </p>
 
 <script>
-$('#formColor').submit(function(event) {
-  var color = $('#formColor input').val();
-  $(document.body).css('background', color);
+if ('serviceWorker' in navigator) {
+  // Register a service worker to allow the tutorial to work offline
+  navigator.serviceWorker.register('/sw.js', {
+    // The scope cannot be parent to the script url
+    scope: '../../'
+  })
+
+  .then(function () {
+    $('.service-worker-status').text('ready!')
+  })
 
```

Comment:
can we access the DOM from https://github.com/hoodiehq/hoodie-camp-tutorial/pull/3/files#diff-7a7a9fab4ca3224cad19254b6aba0be3R55?